__integrations__/index.test.ts

```diff
@@ -261,6 +261,35 @@ describe("Create Message |> Middleware errors", () => {
     });
   });
 
+  it("should return 403 when creating a no ADVANCED third party message with wrong permission", async () => {
+    const nodeFetch = getNodeFetch({
+      "x-user-groups":
+        customHeaders["x-user-groups"] + ",ApiMessageWriteAdvanced"
+    });
```

Comment:
A ThirdPartyMessage with `Standard` level returns 201.
Is this the intended behavior?

Reply:
Plus, tests says `when creating a no ADVANCED..`, but we added `ApiMessageWriteAdvanced` as header.

Reply:
Nope, actually thirdPartyMessage with `Standard` level must return 403 if user doesn't have `ApiThirdPartyMessageWrite` group.

Reply:
Sure, With `ApiThirdPartyMessageWrite` of course (I missed this part, sorry).